stablehlo/dialect/StablehloOps.cpp

```diff
@@ -24,6 +24,7 @@ limitations under the License.
 #include <array>
 #include <cstdint>
 #include <functional>
+#include <iostream>  // FIXME
```

Comment:
What does "FIXME" stand for?

Reply:
Does clang tidy help with this? I can add a comment to #61 to consider if so.

Reply:
Hmm I'm not sure. Your comment has already registered in #61, so we're all good :)